x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel.tsx

```diff
@@ -224,6 +225,7 @@ export function InnerWorkspacePanel({
           className="lnsExpressionRenderer__component"
           padding="m"
           expression={expression!}
+          reload$={plugins.data.query.timefilter.timefilter.getAutoRefreshFetch$()}
```

Comment:
Haven't tested yet, but looks like `timefilter.getAutoRefreshFetch$()` always returns new observable. 
If this is the case, then here: src/plugins/expressions/public/react_expression_renderer.tsx subscribe/unsubscribe is happening during each render call. 
I am not sure if this causes any issues except it adds additional work to perform.